packages/demo/src/AppCCP.svelte

```diff
@@ -241,16 +241,24 @@
 
 <footer>
     <a
-        class="ccp"
-        href="https://dktk.dkfz.de/klinische-plattformen/ueber-die-ccp/about-ccp"
+        class="known-issues"
+        href="https://hub.dkfz.de/s/iP6A7zJzAQya3iC"
         target="_blank"
     >
+        Known Issues
+    </a>
+    <a
+        href="https://dktk.dkfz.de/forschung/Plattformen-und-Technologie-Netzwerke/klinische-plattformen/ccp-faq"
+        class="faq"
+        target="_blank">FAQ</a
+    >
+    <a class="ccp" href="https://dktk.dkfz.de/ccp" target="_blank">
         Clinical Communication Platform (CCP)
     </a>
     <a class="email" href="mailto:CCP@dkfz.de">Kontakt</a>
     <a
         class="user-agreement"
-        href="https://hub.dkfz.de/s/xcGZHda8LHd6yGS"
+        href=" https://hub.dkfz.de/s/MPCg2kK23LH8Yii"
```

Comment:
Can you remove the space infront of the link? Link seems good, but i'm not sure if a space infront will work.

Reply:
done